CoreScriptsRoot/Modules/DeveloperConsoleModule.lua

```diff
@@ -2990,38 +2995,37 @@ local function onDevConsoleVisibilityChanged(isVisible)
 	end
 end
 
-local getDeveloperConsoleIsCreating = false
+local devConsoleCreating = false
```

Comment:
I believe this also stems from the http call, if we can isolate that I think we can make this change a lot smaller